src/main/java/igrad/logic/commands/EditCommand.java

```diff
@@ -1,105 +1,100 @@
-package seedu.address.logic.commands;
+package igrad.logic.commands;
 
+import static igrad.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static igrad.logic.parser.CliSyntax.PREFIX_NAME;
+import static igrad.logic.parser.CliSyntax.PREFIX_PHONE;
+import static igrad.logic.parser.CliSyntax.PREFIX_TAG;
 import static java.util.Objects.requireNonNull;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
-import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
 
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Optional;
 import java.util.Set;
 
-import seedu.address.commons.core.Messages;
-import seedu.address.commons.core.index.Index;
-import seedu.address.commons.util.CollectionUtil;
-import seedu.address.logic.commands.exceptions.CommandException;
-import seedu.address.model.Model;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
-import seedu.address.model.tag.Tag;
+import igrad.commons.core.Messages;
+import igrad.commons.core.index.Index;
+import igrad.commons.util.CollectionUtil;
+import igrad.logic.commands.exceptions.CommandException;
+import igrad.model.Model;
+import igrad.model.module.Email;
+import igrad.model.module.Module;
+import igrad.model.module.Name;
+import igrad.model.module.Phone;
+import igrad.model.tag.Tag;
 
 /**
- * Edits the details of an existing person in the address book.
+ * Edits the details of an existing module in the course book.
```

Comment:
There is some inconsistency with full stops `.` at the back of such `constructor` . I know its been there in the original code-base, would try to fix it.  E.g,


_DeleteCommandParser.java_
```
/**
 * Parses input arguments and creates a new DeleteCommand object
 */
public class DeleteCommandParser implements Parser<DeleteCommand> {
...
```

VS

_EditCommand.java (this current file)_
```
/**
 * Edits the details of an existing person in the address book.
 */
public class EditCommand extends Command {
...
```

Reply:
Sorry i didn't catch this! Each person can probably just look out for this when they are modifying the original code and then we can have a final check at the end :)